### PR TITLE
Support extra classes at page level (close: #84)

### PR DESCRIPTION
**Pull request by ulivz**

### CODE REVIEW

**Comment by ulivz** [comment]
Not know whether it's making sense yet, if not just close it and will not need to add docs for that, Yup

File: lib/default-theme/Layout.vue [hunk 1]
```diff
@@ -63,6 +59,15 @@ export default {
         this.$route,
         this.$site
       )
+    },
+    extraClasses() {
+      const { frontmatter } = this.$page || {}
+      const customClass = frontmatter.class
```
Comment (yyx990803) [inline lib/default-theme/Layout.vue]: maybe `pageClass` is more explicit

**Comment by yyx990803** [comment]
I think this is a decent addition, we need to document this though.

**Comment by ulivz** [comment]
Addressed comment and add docs as well.

**Pull request merged**